src/sentry/snuba/metrics/fields/base.py

```diff
@@ -1824,6 +1825,12 @@ def generate_where_statements(
             snql_func=on_demand_failure_rate_snql_factory,
             default_null_value=0,
         ),
+        DerivedOp(
+            op="on_demand_count_web_vitals",
+            can_orderby=True,
+            snql_func=total_count,
```

Comment:
Why are you using the total query? Don't you have to filter on the `quality` tag?

Reply:
Your right, I thought encoded it in the query hash was sufficient, forgot that it had that data but it also stored that tag. I was thinking the single extracted metric would already represent the total value, since we add the quality to the query hash.
eg.
```"on_demand_count_web_vitals:measurements.lcp:good;{'name': 'event.duration', 'op': 'gte', 'value': 100.0}"```

I suppose the `RuleCondition` would have to be against the entire metric, not just in tags for that to work.